site/source/_posts/2019-11-21-nim-vs-crystal-part-2-threading-tooling.md

```diff
@@ -0,0 +1,302 @@
+title: Nim vs Crystal - Part 2 - Threading & Tooling
+summary: "Crystal and Nim go head-to-head to figure out the best modern, low-level programming language!  In part 2, Threading & Tooling  are reviewed."
+author: robin_percy
+categories:
+  - tutorials
+layout: blog-post
+image: '/assets/images/nim-crystal-header_blank.jpg'
+---
+
+![crystal vs nim](http://embark.status.im/assets/images/nim-crystal-header-img_NEW.jpg)
+
+Welcome back to my series comparing the two sweethearts of the modern low-level programming world.  In [part 1](https://embark.status.im/news/2019/11/18/nim-vs-crystal-part-1-performance-interoperability/), I talked about my views on the interoperability of the two languages, alongside the performance figures of both.  Article #1 managed to throw-up a couple of surprises, but I have to admit; these made it all the more enjoyable to write!
```

Comment:
Same here, the link can just be:`/news/2019/11/18/nim-vs-crystal-part-1-performance-interoperability/`